GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -937,20 +937,17 @@ public void PerformRefreshRevisions(Func<RefsFilter, IReadOnlyList<IGitRef>> get
 
                 _buildServerWatcher.CancelBuildStatusFetchOperation();
 
-                base.Refresh();
-
                 _superprojectCurrentCheckout = null;
-                Subject<IReadOnlyList<GitRevision>> observeRevisions = new();
-                _revisionSubscription?.Dispose();
-                _revisionSubscription = observeRevisions
-                    .ObserveOn(ThreadPoolScheduler.Instance)
-                    .Subscribe(OnRevisionRead, OnRevisionReaderError, OnRevisionReadCompleted);
 
                 IReadOnlyList<ObjectId>? currentlySelectedObjectIds = _gridView.SelectedObjectIds;
                 _gridView.SuspendLayout();
                 _gridView.SelectionChanged -= OnGridViewSelectionChanged;
                 _gridView.ClearSelection();
                 _gridView.Clear();
+
+                // Apply settings early, adapt column widths, etc.
+                Refresh();
```

Comment:
Can this be moved line 1031 so the long running Git commands can be started?
Maybe 5 ms saved in the startup.
The extra call on line 1029 should be changed to _gridView.Invalidate();
Approving anyway, very minor

Reply:
Done